fixtures/chauvet-dj/hurricane-1600.json

```
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Hurricane 1600",
+  "shortName": "Chauvet - Hurricane 1600",
+  "categories": ["Smoke", "Hazer"],
```

```suggestion
  "categories": ["Smoke"],
```

Webpage and manual only mention fog (= smoke), not haze.